pkg/kubelet/apis/kubelet.config.k8s.io/types.go

```diff
@@ -150,43 +145,41 @@ type KubeletConfiguration struct {
 	// garbage collected.
 	ImageMinimumGCAge metav1.Duration
 	// imageGCHighThresholdPercent is the percent of disk usage after which
-	// image garbage collection is always run.
+	// image garbage collection is always run. The percent is calculated as
+	// this field value out of 100.
 	ImageGCHighThresholdPercent int32
 	// imageGCLowThresholdPercent is the percent of disk usage before which
 	// image garbage collection is never run. Lowest disk usage to garbage
-	// collect to.
+	// collect to. The percent is calculated as this field value out of 100.
 	ImageGCLowThresholdPercent int32
 	// How frequently to calculate and cache volume disk usage for all pods
 	VolumeStatsAggPeriod metav1.Duration
-	// KubeletCgroups is the absolute name of cgroups to isolate the kubelet in.
-	// +optional
```

Comment:
this removed intentionally?

Reply:
I don't think +optional makes sense on internal types